README.md

```diff
@@ -32,16 +33,16 @@ const AntiSpam = new DiscordAntiSpam({
   maxDuplicatesWarning: 7, // Amount of same messages sent that will be considered as duplicates that will cause a warning.
   maxDuplicatesBan: 10, // Amount of same messages sent that will be considered as duplicates that will cause a ban.
   deleteMessagesAfterBanForPastDays: 1, // Amount of days in which old messages will be deleted. (1-7)
-  ignoredUsers: [], // array of ignored user ids
-  ignoredGuilds: [] // array of ignored guild ids
-  },
   
+  ignoredUsers: [], // array of ignored user ids
+  ignoredGuilds: [], // array of ignored guild ids
+  exemptUsers: (member) => member.user.createdTimestamp > 15552000000, // Bypass users if his account is more their 6 months old
```

Comment:
The whole library is not meant to ignore uses which accounts that are 6 months old or older. This should not be in the example.